src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -3223,14 +3225,32 @@ private void CreateContainerProcessInternal()
                     // Create PowerShell process inside the container.
                     //
                     result = HcsCreateProcess(ComputeSystem, cmd, ref ProcessInformation, ref Process, ref resultString);
-                    if (result != 0)
+                    if (result == 0)
                     {
-                        processId = 0;
-                        error = result;
+                        processId = Convert.ToInt32(ProcessInformation.ProcessId);
+                    }
+                    else if (result == 2147942402) // 0x80070002 (2147942402) - The system cannot find the file specified.
```

Comment:
Prefer to just have the number on RHS in hex as it's easier to perform a web search against a hex error code.  The comment can then be simplified.